prime-router/metadata/schemas/covid-19.schema

```diff
@@ -9,6 +9,7 @@
 #
 name: covid-19
 topic: covid-19
+trackingElement: message_id
```

Comment:
Why `message_id` vs `specimen_id`?

Reply:
Good catch. I think speciment_id would be the better choice.

Reply:
Went and took a closer look. I had it as message_id because in the covid-19 schema, specimen_id is mapped to message_id.  If I use message_id, I'm going straight to the source.